cmd/bitrot-streaming.go

```diff
@@ -152,7 +153,7 @@ func (b *streamingBitrotReader) ReadAt(buf []byte, offset int64) (int, error) {
 		streamOffset := (offset/b.shardSize)*int64(b.h.Size()) + offset
 		if len(b.data) == 0 && b.tillOffset != streamOffset {
 			b.rc, err = b.disk.ReadFileStream(context.TODO(), b.volume, b.filePath, streamOffset, b.tillOffset-streamOffset)
-			if err != nil {
+			if err != nil && !errors.Is(err, errDiskNotFound) {
```

Comment:
Even if "annoying", this shows real issues that should be addressed by the user. IMO we should keep the logging as-is.

Reply:
> Even if "annoying", this shows real issues that should be addressed by the user. IMO we should keep the logging as-is.

If errDiskNotFound is only returned if a disk is offline (networking issue) - then I guess we can rely on the connectDisks() routine to print those type errors and avoid logging them here.